### PR TITLE
feat(TDI-46737):print jobinfo log

### PR DESCRIPTION
**Pull request by pyzhou-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-46737

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by jzhao-talend — APPROVED** [review]
LGTM

File: main/plugins/org.talend.designer.codegen/jet_stub/footer.javajet [hunk 1]
```diff
@@ -334,6 +336,37 @@
...
+    }
+
+    private java.util.Properties readJobInfo(java.io.File jobInfo){
+	
+        java.util.Properties properties = new java.util.Properties();
+        if(jobInfo.exists()) {
+            try {
+                properties.load(new java.io.FileInputStream(jobInfo));
+            } catch (IOException e) {
+                log.debug("Read jobInfo.properties file fail: " + e.getMessage());
```
Comment (bgunics-talend) [inline main/plugins/org.talend.designer.codegen/jet_stub/footer.javajet:364]: What if log4j is disabled?

File: main/plugins/org.talend.designer.codegen/jet_stub/footer.javajet [hunk 2]
```diff
@@ -334,6 +336,37 @@
...
+            }
+        }
+		log.warn("Project name: {}\nJob name: {}\nGIT Commit ID: {}\nTalend Version: {}",projectName,jobName,properties.getProperty("gitCommitId"), "<%=System.getProperties().getProperty("talend.studio.version")%>");
```
Comment (bgunics-talend) [inline main/plugins/org.talend.designer.codegen/jet_stub/footer.javajet:367]: Should be info level if log4j is enabled.

**Review by bgunics-talend — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.designer.codegen/jet_stub/footer.javajet (hunk 1 above)
Comment (pyzhou-talend) [reply main/plugins/org.talend.designer.codegen/jet_stub/footer.javajet:364]: fixed, thanks

**Review by jzhao-talend — APPROVED** [review]
LGTM

**Pull request merged**